backend/src/plugins/core/admin/email/settings/show/show.service.ts

```diff
@@ -5,22 +5,29 @@ import { Injectable } from "@nestjs/common";
 import { ShowAdminEmailSettingsServiceObj } from "./dto/show.obj";
 
 import { HelpersAdminEmailSettingsService } from "../../helpers.service";
+import { getConfigFile } from "@/config";
 
 @Injectable()
 export class ShowAdminEmailSettingsService extends HelpersAdminEmailSettingsService {
   show(): ShowAdminEmailSettingsServiceObj {
+    const configSettings = getConfigFile();
+
     if (!fs.existsSync(this.path)) {
       return {
         smtp_host: "",
         smtp_user: "",
         smtp_secure: false,
-        smtp_port: 587
+        smtp_port: 587,
+        ...configSettings.settings.email
```

Comment:
should be in separate variable, especially where used again in return statement

Reply:
Show shows everything from email settings so it's fine here.